priv/repo/migrations/20171020100846_add_checkin_date.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Library.Repo.Migrations.AddCheckinDate do
+  use Ecto.Migration
+
+  def change do
+    alter table(:book_loans) do
+      add :checked_in, :naive_datetime
+      remove :queue
```

Comment:
We should probably define `up/down` behaviour for this instead of using `change`. Otherwise it won't know what to rollback to and we'll have errors if we ever need to. There's an example at the top of [`Ecto.Migration` docs](https://hexdocs.pm/ecto/Ecto.Migration.html).